examples/step-2/doc/results.dox

```diff
@@ -40,8 +40,8 @@ and which can't couple when discretizing a local, i.e. differential,
 equation.)
 <table style="width:60%" align="center">
   <tr>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-2.sparsity-1.svg" alt=""></td>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-2.sparsity-2.svg" alt=""></td>
+    <td><img src="https://www.dealii.org/images/steps/developer/step-2.sparsity-1.svg" style="width:100%" alt=""></td>
+    <td><img src="https://www.dealii.org/images/steps/developer/step-2.sparsity-2.svg" style="width:100%" alt=""></td>
```

Comment:
Is 100% the right width? Does that mean 50% of the page width with two columns?

Reply:
No, it is 100% of the table cell they are in. The table has a total size set two lines above.